db/schema.rb

```diff
@@ -694,6 +706,7 @@
     t.datetime "session_end_time"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.bigint "process_id"
```

Comment:
Do we need this?

Reply:
No I don't think so.. I'll remove that from my local schema file and commit.